.github/config/rosa-on-demand/distribution-team/external-secret.yaml

```diff
@@ -21,3 +21,20 @@ spec:
     remoteRef:
       key: ci-distro-ultrawombat-com
       property: tls.key
+---
+# clone the integration tests secrets
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: external-secret-integration-test
+  namespace: camunda-platform
+spec:
+  refreshInterval: 1h
+  secretStoreRef:
+    kind: SecretStore
+    name: k8s-store-token-auth
+  target:
+    name: integration-test
```

Comment:
we might need to change this to `integration-test-credentials` to be in line with recent changes on main.